async_upnp_client/ssdp_listener.py

```diff
@@ -43,12 +43,19 @@
     "location",  # Location-header is handled differently!
 }
 
+_INVALID_LOCATIONS = (
+    "://127.0.0.",
```

Comment:
I think this should be `"://127."`, as the whole `127.0.0.0/8` range is regarded as host local.